src/components/__snapshots__/FeeIcon.test.tsx.snap

```diff
@@ -54,9 +54,9 @@ exports[`ExchangeFeeIcon renders correctly 1`] = `
   >
     <svg
       fill="none"
-      height={12}
```

Comment:
i wonder why this changed?

Reply:
It appears to be the default sizing that changes I checked a few icons and they looked the same.

<img src="https://github.com/user-attachments/assets/a0445c6d-7c92-499a-8b25-5d5d3218cf07" width="40%" height="auto" />